servers/rendering/renderer_rd/forward_clustered/render_fc.h

```diff
@@ -32,12 +32,17 @@
 #define RENDERING_SERVER_SCENE_RENDER_FORWARD_CLUSTERED_H
 
 #include "core/templates/paged_allocator.h"
+#include "servers/rendering/renderer_rd/forward_clustered/scene_shader_fc.h"
 #include "servers/rendering/renderer_rd/pipeline_cache_rd.h"
 #include "servers/rendering/renderer_rd/renderer_scene_render_rd.h"
 #include "servers/rendering/renderer_rd/renderer_storage_rd.h"
 #include "servers/rendering/renderer_rd/shaders/scene_forward_clustered.glsl.gen.h"
 
-class RendererSceneRenderForwardClustered : public RendererSceneRenderRD {
+namespace RendererSceneRenderForwardClustered {
+
+class RenderFC : public RendererSceneRenderRD {
```

Comment:
maybe the namespace could just be RendererSceneRenderImplementation

and the file and namespace just ForwardClustered

Reply:
I'm for that, my only concern is that with multiple implementations that are very similar it's harder to find faults.

For instance now with a different namespace for ForwardClustered and ForwardMobile, if I accidentially use `SceneShaderFC` instead of `SceneShaderFM` I get an immediate compile error. If they are all in the same namespace its very easy to make that typo and not notice it.

Reply:
I think tis not a problem if you write SceneShaderForwardClustered and SceneShaderForwardMobile its still not confusing

Reply:
Yeah I can name them fully written out. It will stand out more easily when they are like that. 
Ok let me make those changes.